public/js/posts/show.js

```diff
@@ -2,10 +2,13 @@
 
 {
     // ==== Env ====
-    const token = document.querySelector('meta[name="csrf-token"]').getAttribute('content');
     const hostUrl = 'http://localhost:8000';
```

Comment:
make hostUrl dynamic for cases when we want to deploy your project in heroku or other servers

Reply:
I make it dynamic.